app/services/katello/pulp3/file_unit.rb

```diff
@@ -0,0 +1,44 @@
+module Katello
+  module Pulp3
+    class FileUnit < PulpContentUnit
+      include LazyAccessor
+      CONTENT_TYPE = "iso".freeze
+
+      lazy_accessor :pulp_facts, :initializer => :backend_data
+
+      def self.ids_for_repository(repo_id)
+        repo = Katello::Pulp3::Repository::File.new(Katello::Repository.find(repo_id), SmartProxy.pulp_master)
+        repo_content_list = repo.content_list
+        repo_content_list.map { |content| content.try(:_href) }
+      end
+
+      def self.pulp_data(href)
+        content_unit = SmartProxy.pulp_master!.pulp3_api.content_file_files_read(href)
+        content_unit.as_json
+      end
+
+      def update_model(model)
+        custom_json = {}
+        custom_json['checksum'] = backend_data['sha256']
```

Comment:
Based on this https://pulp.plan.io/issues/4827 , we should be able to pick up sha 256 once the bindings are updated.